test/bridge.test.ts

```diff
@@ -803,6 +882,13 @@ describe("Bridge", async function () {
     expect(flushL2Response.consumed_messages.from_l1).to.be.empty;
     expect(flushL2Messages).to.have.a.lengthOf(1);
 
+    //check that there are enough rewards on l1 bridge
+    const bridgePendingRewards = await incentives.getRewardsBalance(
```

Comment:
we could use `bridgePendingRewards` in line 876 as argument of the function `bridge_rewards`. then we are sure that incentives controller can provide enough rewards to make the transaction pass. that is:

```js
    await l2user.invoke(l2Bridge, "bridge_rewards", {
      l1_recipient: BigInt(l1user.address),
      amount: { high: 0n, low: bridgePendingRewards },
    });
```

Reply:
bridgePendingRewards=all rewards that are claimable by the bridge! user2 only has a fraction of the pie :)

Reply:
indeed :) we could then use `min(bridgePendingRewards, l2ClaimedRewards)`